sippy-ng/src/component_readiness/CompReadyTestReport.js

```diff
@@ -504,87 +438,41 @@ View the test details report at ${document.location.href}
-            loadedParams.variantCrossCompare,
-            loadedParams.includeVariantsCheckedItems
-          )}
-        </Grid>
-        <Grid item xs={6}>
-          {printParamsAndStats(
-            'Sample (being evaluated)',
-            data.sample_stats,
-            loadedParams.sampleStartTime.toString(),
-            loadedParams.sampleEndTime.toString(),
-            loadedParams.variantCrossCompare,
-            loadedParams.compareVariantsCheckedItems
-          )}
-        </Grid>
-      </Grid>
-      <div style={{ marginTop: '10px', marginBottom: '10px' }}>
-        <label>
-          <input
-            type="checkbox"
-            checked={showOnlyFailures}
-            onChange={handleFailuresOnlyChange}
-          />
-          Only Show Failures
-        </label>
-      </div>
-      <TableContainer component="div" className="cr-table-wrapper">
-        <Table className="cr-comp-read-table">
-          <TableHead>
-            <TableRow>
-              {tableCell('ProwJob Name', 0)}
-              {tableCell('Basis Info', 1)}
-              {tableCell('Basis Runs', 2)}
-              {tableCell('Sample Info', 3)}
-              {tableCell('Sample Runs', 4)}
-              {tableTooltipCell(
-                'Statistically Significant',
-                5,
-                significanceTitle
-              )}
-            </TableRow>
-          </TableHead>
-          <TableBody>
-            {/* Ensure we have data before trying to map on it; we need data and rows */}
-            {data && data.job_stats && data.job_stats.length > 0 ? (
-              data.job_stats
-                .sort((a, b) => {
-                  if (a.significant && b.significant) {
-                    return 0
-                  } else if (a.significant) {
-                    // This makes it so that statistically significant ones go to the top.
-                    return -1
-                  } else {
-                    return 1
-                  }
-                })
-                .map((element, idx) => {
-                  return (
-                    <CompReadyTestDetailRow
-                      key={idx}
-                      element={element}
-                      idx={idx}
-                      showOnlyFailures={showOnlyFailures}
-                    ></CompReadyTestDetailRow>
-                  )
-                })
-            ) : (
-              <TableRow>
-                {/* No data to render (possible due to a Cancel */}
-                <TableCell align="center">No data ; reload to retry</TableCell>
-              </TableRow>
-            )}
-          </TableBody>
-        </Table>
-      </TableContainer>
+      {isBaseOverride ? (
+        <Fragment>
+          <Tabs
+            value={activeTabIndex}
+            onChange={handleTabChange}
+            aria-label="Test Report Tabs"
+          >
+            <Tab label="Basis Tests" {...tabProps(0)} />
+            <Tab label="Override Tests" {...tabProps(1)} />
```

Comment:
These titles are too ambiguous for a newb. It's admittedly difficult to come up with better. Maybe:
```suggestion
            <Tab label="Severest Result" {...tabProps(0)} />
            <Tab label="Specified Result" {...tabProps(1)} />
```
Again some kind of explanation for newbs would be helpful... though I'm not sure tooltips work well on tabs

Reply:
I haven't looked into the tooltip yet but how does this look:

![image](https://github.com/user-attachments/assets/408c530d-547b-456a-a933-fda48f69b4e2)

Reply:
I like this one better. I was confused with the terms as well. But this makes it clear.